example/mysql/views/layouts/main.tt

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<title>Management of company users</title>
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.4/css/bootstrap.min.css" integrity="sha384-2hfp1SzUoho7/TsGGGDaFdsuuDL0LX2hnUp6VkX3CUQ2K4K+xjboZdsXyp4oUHZj" crossorigin="anonymous">
+<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+<meta http-equiv="x-ua-compatible" content="ie=edge">
+</head>
+<body>
+
+<nav class="navbar navbar-light bg-faded">
+
+<!-- Toggle Button -->
+<button class="navbar-toggler hidden-sm-up" type="button" data-toggle="collapse" data-target="#nav-content">
+&#9776;
+</button>
+
+<!-- Nav Content -->
+<div class="collapse navbar-toggleable-xs" id="nav-content">
+<a class="navbar-brand" href="#">Manage company users</a>
+<ul class="nav navbar-nav">
+<li class="nav-item">
+<a class="nav-link" href="/people">Home</a>
+</li>
+<li class="nav-item">
+<a class="nav-link" href="/people/add">Add</a>
```

Comment:
can we parameterize  "Manage company user" titles and the /people links so they works correctly for other simple_crud() uses? Ideally this template could be used for other simple_crud calls out of the box.